R/retry.R

```diff
@@ -16,6 +20,10 @@
 #'   \code{pause_cap} seconds.
 #' @param quiet If \code{FALSE}, will print a message displaying how long
 #'   until the next request.
+#' @param terminate_on Optional vector of numeric or integer HTTP status codes
```

Comment:
Integer is a sub-type of numeric, so you don't need to mention both here (or test for both below)